controller/src/test/java/io/pravega/controller/server/eventProcessor/ConcurrentEventProcessorTest.java

```diff
@@ -75,38 +92,40 @@ public PositionInternal asImpl() {
         }
     }
 
-    ConcurrentEventProcessor<TestEvent, TestRequestHandler> processor;
-    int checkpoint = -1;
-    Map<Integer, Boolean> map = new HashMap<>();
-    CompletableFuture<Void> latch = new CompletableFuture<>();
-    CompletableFuture<Void> result = new CompletableFuture<>();
+    private int checkpoint = -1;
+    private Map<Integer, Boolean> map = new HashMap<>();
```

Comment:
Also note that even if the test does not involve threads, putting a timeout on a test will cause a thread to be spawned and the test to be run in that thread which means member variables that are initialized inline like this are problematic.